Paraswap-Agents/delegate-calls/src/finding.ts

```diff
@@ -0,0 +1,15 @@
+import { Finding, FindingType, FindingSeverity } from "forta-agent";
+
+export const createFinding = (multiPathAddress: string) => {
+  return Finding.fromObject({
+    name: "MultiPath DelegateCall",
+    description: "A delegated call has been made to the MultiPath contract",
+    alertId: "PARASWAP-3",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    protocol: "Paraswap",
+    metadata: {
+      multiPathAddress: multiPathAddress
```

Comment:
this is not a deal but just for convenience, since the and the value have the same name, it could also be written like this:
```
metadata: {
  multiPathAddress
}
```